dpe/src/commands/rotate_context.rs

```diff
@@ -51,7 +69,9 @@ impl CommandExecution for RotateCtxCmd {
         if self.uses_target_is_default() {
             let default_context_idx =
                 dpe.get_active_context_pos(&ContextHandle::default(), locality);
-            if default_context_idx.is_ok() {
+            if default_context_idx.is_ok()
+                || self.non_default_valid_handles_exist(dpe, locality, idx)
+            {
```

Comment:
If there is a default context there shouldn't be any other contexts in the locality,  so in this scenario if `default_context_idx` is valid and `idx` is valid, they must be the same index. So the request is to turn the default context into the default context. I'm leaning toward still considering this an error, but what do you think?

Reply:
I agree especially since the spec says "If TARGET_IS_DEFAULT is set, NEW_CONTEXT_HANDLE SHALL be made the default in the caller's locality. If the default context handle is already defined in the caller's locality, this command SHALL fail.", but we are already doing this since default_context_idx.is_ok() would be true, right?

Reply:
Okay cool LGTM!

Reply:
Just reopening this comment until the PR is updated

Reply:
Never mind, I misread this.